hedera-node/src/test/java/com/hedera/services/tokens/HederaTokenStoreTest.java

```diff
@@ -1053,6 +1061,7 @@ public void updateHappyPathIgnoresZeroExpiry() {
 	public void updateHappyPathWorksForEverythingWithNewExpiry() {
 		// setup:
 		subject.symbolKeyedIds.put(symbol, misc);
+		subject.addKnownTreasury(treasury, misc);
```

Comment:
you should add checks for the known treasuries at the bottom of this test as-well. You can verify that the new treasury is added and the old one is removed